docs/getting-started/supported-languages-and-tools.md

```diff
@@ -456,16 +504,18 @@ The table below lists all languages and frameworks that Codacy supports and the
       <td><a href="https://trivy.dev">Trivy</a></td>
       <td>-</td>
       <td>-</td>
+      <td>-</td>
     </tr>
   </tbody>
 </table>
 
 <sup><span id="yaml-only">1</span></sup>: Currently, Trivy only supports scanning YAML files on this platform.  
 <sup><span id="client-side">2</span></sup>: Supported as a [client-side tool](../repositories-configure/local-analysis/client-side-tools.md).  
-<sup><span id="cppcheck-misra">3</span></sup>: Currently, Cppcheck only supports [checking the MISRA guidelines for C](https://cppcheck.sourceforge.io/misra.php).  
-<sup><span id="dart-limitations">4</span></sup>: Currently, Codacy only supports including the packages [lints](https://pub.dev/packages/lints) and [<span class="skip-vale">flutter_lints</span>](https://pub.dev/packages/flutter_lints) on dartanalyzer configuration files.  
-<sup><span id="ruby-31">5</span></sup>: Currently, Codacy doesn't support any static code analysis tool for [Ruby 3.1](https://www.ruby-lang.org/en/news/2021/12/25/ruby-3-1-0-released/).  
-<sup><span id="swiftlint-complexity">6</span></sup>: Supports [reporting warnings or errors](https://realm.github.io/SwiftLint/cyclomatic_complexity.html) on functions above specific complexity thresholds. Enable the rule **Cyclomatic Complexity** on the [Code patterns page](../repositories-configure/configuring-code-patterns.md), or use a [configuration file](https://realm.github.io/SwiftLint/index.html#configuration) to customize the thresholds.  
+<sup><span id="any-path">3</span></sup>: These files are scanned regardless of their file path.  
```

Comment:
Not sure what you mean by this but the current logic is to let Trivy analyse all files but only report issues with line numbers for files that Codacy told Trivy to analyse.

For example, if the dependencies file is ignored on Codacy, no issues will be reported for it.

Reply:
On second thought, this info is not vital. I'll remove it for now and reevaluate if we get any complaints.